packages/slice-machine/tests/server/env.test.js

```diff
@@ -211,59 +211,4 @@ describe("getEnv", () => {
     const { env } = await getEnv(TMP);
     expect(env.framework).toEqual("vanillajs");
   });
-
-  test("it should not consider the user logged in if the base from .prismic is different than the one in SM.json", async () => {
```

Comment:
probably worth keeping one of these to make sure the cookie is loaded (not sure if that's covered else where)